fields/types/email/EmailField.js

```diff
@@ -13,8 +13,8 @@ module.exports = Field.create({
 	
 	renderValue: function() {
 		return this.props.value
-			? <a className="ui-related-item" href={"mailto:" + this.props.value}>{this.props.value}</a>
-			: <div className="help-block">(not set)</div>;
+			? <a className='ui-related-item' href={'mailto:' + this.props.value}>{this.props.value}</a>
+			: <div className='help-block'>(not set)</div>;
```

Comment:
JSX attributes should be `"`, javascript strings should be `'`.

`className="help-block"` is a JSX attribute
`href={'mailto:' + this.props.value}` contains a javascript string